config/bidderinfo.go

```diff
@@ -356,6 +358,50 @@ func validateMaintainer(info *MaintainerInfo, bidderName string) error {
 	return nil
 }
 
+func validateAliasCapabilities(bidder BidderInfo, infos BidderInfos, bidderName string) error {
+	if len(bidder.AliasOf) == 0 {
+		return nil
+	}
+
+	parentBidder, parentFound := infos[bidder.AliasOf]
+	if !parentFound {
+		return fmt.Errorf("bidder: %s not found for an alias: %s", bidder.AliasOf, bidderName)
+	}
+
+	if bidder.Capabilities != nil {
+		if parentBidder.Capabilities == nil {
+			return fmt.Errorf("capabilities for alias: %s should be a subset of capabilities for parent bidder: %s", bidderName, bidder.AliasOf)
+		}
+
+		if bidder.Capabilities.Site != nil && parentBidder.Capabilities.Site != nil {
+			return validateAliasPlatformInfo(*parentBidder.Capabilities.Site, *bidder.Capabilities.Site, bidderName, bidder.AliasOf)
+		}
+
+		if bidder.Capabilities.App != nil && parentBidder.Capabilities.App != nil {
+			return validateAliasPlatformInfo(*parentBidder.Capabilities.App, *bidder.Capabilities.App, bidderName, bidder.AliasOf)
+		}
+
+		return fmt.Errorf("capabilities for alias: %s should be a subset of capabilities for parent bidder: %s", bidderName, bidder.AliasOf)
+	}
+
+	return nil
+}
+
+func validateAliasPlatformInfo(parentInfo PlatformInfo, aliasInfo PlatformInfo, bidderName string, parentBidderName string) error {
```

Comment:
nitpick - might wanna consider renaming this function to `isAliasPlatformInfoSubsetOfParent`